modules/services/preferences/src/main/java/au/com/shiftyjelly/pocketcasts/preferences/Settings.kt

```diff
@@ -522,16 +522,16 @@ interface Settings {
     fun getCloudSortOrder(): CloudSortOrder
     fun getCloudAddToUpNext(): Boolean
     fun setCloudAddToUpNext(value: Boolean)
-    fun getCloudDeleteAfterPlaying(): Boolean
-    fun setCloudDeleteAfterPlaying(value: Boolean)
+    fun getDeleteLocalFileAfterPlaying(): Boolean
+    fun setDeleteLocalFileAfterPlaying(value: Boolean)
+    fun getDeleteCloudFileAfterPlaying(): Boolean
+    fun setDeleteCloudFileAfterPlaying(value: Boolean)
     fun getCloudAutoUpload(): Boolean
     fun setCloudAutoUpload(value: Boolean)
     fun getCloudAutoDownload(): Boolean
     fun setCloudAutoDownload(value: Boolean)
     fun getCachedSubscription(): SubscriptionStatus?
     fun setCachedSubscription(subscriptionStatus: SubscriptionStatus?)
-    fun getCloudDeleteCloudAfterPlaying(): Boolean
-    fun setCloudDeleteCloudAfterPlaying(value: Boolean)
```

Comment:
I found the previous names for these settings to be super confusing. These changes seemed clearer to me. Let me know if you think something else would be better.

Reply:
Looks fine to me.